start.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/bash
-for d in */ ; do
-    if [ "$d" != "node_modules/" ]
+if [ $# -gt 0 ]
+then
+  for dir in "$@"; do
+    if [ "$dir" != "node_modules" ] && [ "$dir" != "commons" ]
     then
-        echo "${d}"
-        testcafe chrome "${d}index.js"
+      echo "${dir}/"
+      testcafe chrome "${dir}/index.js"
```

Comment:
To już takie w ogóle by było zarąbiste, gdybyś przeglądarkę też mógł sobie przełącznikiem jakimś podać. Na przykład `./start.sh --browser chrome`, ale pewnie nigdy nie będziemy korzystać z czegokolwiek innego, niż chrome XD